extensions/blocks/conversation/save.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+export default function save ( { attributes } ) {
+	const baseClassName = 'wp-block-jetpack-conversation';
+
+	return (
+		<div
+			className={ classnames(
+				baseClassName,
+				'entry-content',
```

Comment:
Wouldn't this inherit styles defined by themes that are not intended for a block?

Think of themes that set a border and a box-shadow to the `entry-content` class. I guess we don't want these styles in our block.

Reply:
I think we should handle styles and themes in other PR. Definitely, this is something that we want to implement properly. We are not adding color implementation here for the same reason.